docs/api/puppeteer.connection.createsession.md

```diff
@@ -8,7 +8,7 @@ sidebar_label: Connection.createSession
 
 ```typescript
 class Connection {
-  createSession(targetInfo: Protocol.Target.TargetInfo): Promise<CDPSession>;
```

Comment:
We need to revert this change, as we don't expose `CdpCDPSession` publicly so there will be no way for user using Puppeteer search for it.

Reply:
Ah, thanks for the context - then I guess indeed changing anything isn't worth it unless we want to explore exposing CdpCDPSession 🤔

Reply:
On the other hand - I'll be back in a few :)

Reply:
Reverted